src/Discord.Net.WebSocket/Entities/Users/SocketUser.cs

```diff
@@ -37,30 +37,33 @@ internal virtual bool Update(ClientState state, Model model)
             {
                 var newVal = ushort.Parse(model.Discriminator.Value);
                 if (newVal != DiscriminatorValue)
-                { 
+                {
                     DiscriminatorValue = ushort.Parse(model.Discriminator.Value);
                     hasChanges = true;
                 }
             }
             if (model.Bot.IsSpecified && model.Bot.Value != IsBot)
-            { 
+            {
                 IsBot = model.Bot.Value;
                 hasChanges = true;
             }
             if (model.Username.IsSpecified && model.Username.Value != Username)
-            { 
+            {
                 Username = model.Username.Value;
                 hasChanges = true;
             }
             return hasChanges;
-        } 
+        }
 
         public async Task<IDMChannel> GetOrCreateDMChannelAsync(RequestOptions options = null)
             => GlobalUser.DMChannel ?? await UserHelper.CreateDMChannelAsync(this, Discord, options) as IDMChannel;
 
         public string GetAvatarUrl(ImageFormat format = ImageFormat.Auto, ushort size = 128)
             => CDN.GetUserAvatarUrl(Id, AvatarId, size, format);
 
+        public string GetDefaultAvatarUrl()
+            => CDN.GetDefaultUserAvatarUrl(ushort.Parse(Discriminator));
```

Comment:
`DiscriminatorValue` applies here also.

Reply:
didn't realize that there was already a ushort value.
change added to PR